docs/sources/next/javascript-api/k6-experimental/browser/element-handle/dblclick.md

```diff
@@ -0,0 +1,58 @@
+---
+title: 'dblclick([, options])'
+excerpt: 'Browser module: elementHandle.dblclick([, options]) method'
+---
+
+# dblclick([, options])
+
+Mouse double clicks the element.
+
+<TableWithNestedRows>
+
+| Parameter           | Type     | Default | Description                                                                                                                                                                                                                                                                                                                                   |
+| ------------------- | -------- | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| options             | object   | `null`  |                                                                                                                                                                                                                                                                                                                                               |
+| options.button      | string   | `left`  | The mouse button (`left`, `middle` or `right`) to use during the action.                                                                                                                                                                                                                                                                      |
+| options.delay       | number   | `0`     | Milliseconds to wait between `mousedown` and `mouseup`.                                                                                                                                                                                                                                                                                       |
+| options.force       | boolean  | `false` | Setting this to `true` will bypass the actionability checks (`visible`, `stable`, `enabled`).                                                                                                                                                                                                                                                 |
+| options.modifiers   | string[] | `null`  | `Alt`, `Control`, `Meta` or `Shift` modifiers keys pressed during the action. If not specified, currently pressed modifiers are used.                                                                                                                                                                                                         |
```

Comment:
What does "currently pressed modifiers" in this context mean?

> If not specified, currently pressed modifiers are used.

Reply:
If the script has used `keyboard.press('Alt')` before `elementHandle.dblclick()` is used then it will take into account the `Alt` key press.